### PR TITLE
Update documentation to improve visibility and avoid duplicates with Grafana documentation

### PR DESCRIPTION
**Pull request by AgnesToulet**
This PR does the following documentation update:
- Move the content of the `remote_rendering_using_docker.md` file to improve its visibility (mainly in the new image rendering configuration section in Grafana documentation).
- Move some installation instructions from Grafana documentation to the plugin main README.md (so it appears here: https://grafana.com/grafana/plugins/grafana-image-renderer/).

To be merged with this OSS PR: https://github.com/grafana/grafana/pull/38759

### CODE REVIEW

**Review by pianohacker — APPROVED** [review]
Looks good to me.

**Review by mjseaman — APPROVED** [review]
LGTM!

File: README.md
Comment (achatterjee-grafana) [inline README.md:28]:
```suggestion
### Grafana CLI
```

File: README.md
Comment (achatterjee-grafana) [reply README.md:36]: Please remove note about Grafana v6.4, it is a very old version.

File: README.md
Comment (achatterjee-grafana) [inline README.md:38]:
```suggestion
This plugin is not compatible with the current Grafana Docker image and requires additional system-level dependencies. We recommend setting up another Docker container for rendering and using remote rendering instead. For instruction, refer to [Run in Docker](#run-in-docker). 
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:40]:
```suggestion
If you still want to install the plugin with the Grafana Docker image, refer to the instructions on building a custom Grafana image in [Grafana Docker documentation](https://grafana.com/docs/installation/docker/#custom-image-with-grafana-image-renderer-plugin-pre-installed).
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:52]:
```suggestion
Grafana Docker images are published at [Docker Hub](https://hub.docker.com/r/grafana/grafana-image-renderer).
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:54]:
```suggestion
The following example shows how you can run Grafana and the remote HTTP rendering service in two separate Docker containers using Docker Compose.
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:56]:
```suggestion
1. Create a `docker-compose.yml` with the following content:
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:76]:
```suggestion
1. Next, run docker compose.
```

File: README.md
Comment (achatterjee-grafana) [inline README.md:116]:
```suggestion
For troubleshooting help, refer to
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: README.md
Comment (AgnesToulet) [inline README.md:18]:
```suggestion
[Grafana Image Rendering documentation](https://grafana.com/docs/image-rendering/).
```

File: README.md
Comment (AgnesToulet) [reply README.md:20]: The top section is already "Requirements", you can't see it in the PR review tab as it already existed and I didn't make any change to this top section. You can see it here: https://github.com/grafana/grafana-image-renderer#requirements

File: README.md
Comment (AgnesToulet) [reply README.md:26]: Applied this with some updates as I had trouble understanding the last paragraph.

File: README.md
Comment (AgnesToulet) [reply README.md:36]: Done.

File: README.md
Comment (AgnesToulet) [reply README.md:28]: Done, with an update to point to the recommended way to do it.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM, if we need any additional change, we can do it later. Thank you!

**Comment by AgnesToulet** [comment]
Great, thanks! I'll wait for the Grafana PR to be approved to merge both PRs around the same time (as some contents were moved between the two docs and there are a lot of links between these docs).

**Pull request merged**
